joomla/joomla-cms
# [com_finder] Add/Improve search tooltip in all views

#### Summary of Changes

Simple PR to add the search tooltip and element label (accessibility) in com_finder.
#### Testing Instructions

Code review, or:
1. Apply patch
2. Go to Components -> Smart Search -> Indexed Content, check the tooltip in the search field.
3. Go to Components -> Smart Search -> Content Maps, check the tooltip in the search field.
4. Go to Components -> Smart Search -> Search Filters, check the tooltip in the search field.
